features/image-transformations/resize-crop-and-other-transformations.md

```diff
@@ -461,6 +508,77 @@ Quite distinctly from the [auto smart crop](resize-crop-and-other-transformation
 **Note:** Smart crop may not give accurate results for some images. This is partially a trade off between speed (needed for real-time transformations) and accuracy.
 {% endhint %}
 
+### Object aware cropping
+
+To automatically provide high focus to a particular object while cropping, you can set the `fo` value to an object from the [supported list](#supported-object-list). This allows for intelligent cropping of the image based on the specific objects detected within it.
+
+It can be used along with [maintain ratio](#examples---object-aware-cropping-maintain-ratio) or [extract crop](#examples---object-aware-cropping-cm-extract) to intelligently crop or extract parts of the image as needed. Learn more from the different examples shown in respective sections.
+
+As discussed below, we can choose not to provide dimensions and deliver an image that is tightly cropped to the object.
+
+#### Tightly crop object
+
+To deliver an image that is tightly cropped to the object, provide `fo-<object name>` without specifying `height` and `width` dimensions. This will return an image that contains only the given object.
+
+
+We can also specify `aspect ratio` along with the `fo-<object name>` focus option without providing `height` and `width`. This keeps the object in focus but may include more of the image to fit the specified `aspect ratio`.
+
+{% tabs %}
+{% tab title="Original" %}
+URL - [https://ik.imagekit.io/a1yisxurxo/car.jpg](https://ik.imagekit.io/a1yisxurxo/car.jpg)
+
+![Original 1920x1280 image](https://ik.imagekit.io/a1yisxurxo/car.jpg)
+{% endtab %}
+
+{% tab title="Tightly cropped image" %}
+URL - [https://ik.imagekit.io/a1yisxurxo/car.jpg?tr=fo-car](https://ik.imagekit.io/a1yisxurxo/car.jpg?tr=fo-car)
+
+![982x301 image](https://ik.imagekit.io/a1yisxurxo/car.jpg?tr=fo-car)
+
+The output is an image containing just car.
+{% endtab %}
+
+{% tab title="Tightly cropped image with aspect ratio" %}
+URL - [https://ik.imagekit.io/a1yisxurxo/car.jpg?tr=fo-car,ar-2_1](https://ik.imagekit.io/a1yisxurxo/car.jpg?tr=fo-car,ar-2_1)
+
+![982x491 image](https://ik.imagekit.io/a1yisxurxo/car.jpg?tr=fo-car,ar-2_1)
+
+The output is an image containing only the car, plus the additional `height` needed to maintain an `aspect ratio` of 2.
+{% endtab %}
+{% tabs %}
+
+We can also crop out a particular object of interest out of multiple different objects present in an image.
```

Comment:
Isn't this example already covered above?

Reply:
This example should be merged with the above one. Same example can easily illustrate the two concepts (tight cropping and picking particular object out of multiple)